core/include/traccc/definitions/primitives.hpp

```diff
@@ -8,12 +8,16 @@
 #pragma once
 
 #if ALGEBRA_PLUGINS_INCLUDE_ARRAY
+#include "detray/plugins/algebra/array_definitions.hpp"
```

Comment:
I wonder if you can include detray definitions headers in traccc definition headers rather than here?

Reply:
That indeed sounds like a better setup... :thinking: